error with NGEO = FALSE

several problem when the option NGEO = FALSE is set:

when building the project, there is an openlayers 3 dependancy error.
since we dont want ngeo, we shouldnt have any openlayers 3 dependancy AT ALL!

```
mkdir -p .build/
touch .build/node_modules.timestamp
INSTANCE_ID=ochngeo APACHE_ENTRY_POINT=/ochngeo/ DEVELOPMENT=TRUE PACKAGE=demo .build/venv/bin/c2c-template --vars vars_demo.yaml --engine mako --files project.yaml.mako

module.js:340
    throw err;
          ^
Error: Cannot find module 'openlayers/node_modules/closure-util'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at [eval]:1:22
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:456:26)
    at evalScript (node.js:536:25)
    at startup (node.js:80:7)
    at node.js:906:3
ERROR when running the expression ''process.stdout.write(require("openlayers/node_modules/closure-util").getLibraryPath())'': Command '['node', '-e', 'process.stdout.write(require("openlayers/node_modules/closure-util").getLibraryPath())']' returned non-zero exit status 8
CONST_Makefile:426: recipe for target 'project.yaml' failed
make: *** [project.yaml] Error 1
```

"solved" (workaround) by running `npm install`

but in the end the project doesnt work at all, because we have 404 on the files:

http://geomapfish-demo.camptocamp.net/ochngeo/wsgi/proj-ngeo/bbe52a7c29aa4d72a95d3027ea0e1c87/build/main.min.css

http://geomapfish-demo.camptocamp.net/ochngeo/wsgi/proj-ngeo/bbe52a7c29aa4d72a95d3027ea0e1c87/build/main.js

--- a/c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl
+++ b/c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl
@@ -361,3 +361,4 @@ interpreted:
         vars:
         - closure_library_path
         cmd: ["node", "-e"]
+        ignore_error: true

